File: Modul-4/C/PRAK405_2410817210001_MuhammadFadillah.c
#include <stdio.h>

int main() {
    int n, kelipatan;
    
    printf("Masukkan jumlah baris dan kelipatan : ");
    scanf("%d %d", &n, &kelipatan);
    
    int total = 0; 

    for (int i = 1; i <= n; i++) {
        int sum = 0; 
        
        for (int j = i; j >= 1; j--) {
            sum += j * kelipatan;
            
            printf("(%d * %d)", j, kelipatan);
            if (j > 1) {
                printf(" + ");
            }
        }

        printf(" = %d\n", sum);
        
        total += sum;
    }
    
    printf("%d\n", total);
    return 0;
}

File: Modul-4/Python/PRAK405_2410817210001_MuhammadFadillah.py
n, kelipatan = map(int, input("Masukkan jumlah baris dan kelipatan : ").split())

total = 0  

for i in range(1, n + 1):
    sum_row = 0  
    row_output = [] 
    
    for j in range(i, 0, -1):
        sum_row += j * kelipatan
        row_output.append(f"({j} * {kelipatan})")
    
    print(" + ".join(row_output), f"= {sum_row}")
    
    total += sum_row

print(total)
